Yoast/yoast-components
# Improve keyword input focus style and contrast ratio

## Summary

This PR can be summarized in the following changelog entry:

- Improves the keyword input field focus style and contrast ratio

## Test instructions
- on the standalone app, yarn start go to localhost:3333
- go to the "Keyword" tab 
- verify the focus style of the keyword input field (without and with remove icon) work correctl
- verify the remove icon has a focus style

<img width="650" alt="screen shot 2018-08-09 at 18 52 09" src="https://user-images.githubusercontent.com/1682452/43914126-42655872-9c07-11e8-9011-6503c960512a.png">

Note: on the standalone app the fields will show the native browser focus style: blue outline on webkit and none on Firefox. 

Also, verify the `.had-error` CSS class works properly when there's an error (you can alter the code or add the CSS class in your browser inspector:

<img width="326" alt="screen shot 2018-08-09 at 19 04 43" src="https://user-images.githubusercontent.com/1682452/43914076-1c8d8d4a-9c07-11e8-9efa-18e764fe4efa.png">

To test the actual focus style in the metabox:
- yarn link yoast-components on the premium plugin
- build the js in the premium folder and in the root
- edit a post, go to the plugin meta box
- verify the focus keyword field has a nice focus style
- verify the additional focus keyword field has a nice focus style
- verify the remove icon button has a focus style

Fixes #706
